### PR TITLE
Update rest_command.markdown configuration

### PR DESCRIPTION
**Pull request by hmmbob**
**Description:**
Update style of REST Command documentation to follow new configuration variables description.
Related to #6385

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** n/a

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/rest_command.markdown [hunk 1]
```diff
@@ -29,15 +29,46 @@ rest_command:
 
 Configuration variables:
```
Comment (frenck) [inline source/_components/rest_command.markdown]: This header should be removed.

File: source/_components/rest_command.markdown [hunk 2]
```diff
@@ -29,15 +29,46 @@ rest_command:
...
-  - **url** (*Required*): The URL (support template) for sending request.
-  - **method** (*Optional*): HTTP method to use (`get`, `post`, `put`, or `delete`). Defaults to `get`.
-  - **headers** (*Optional*): The headers for the requests.
-  - **payload** (*Optional*): A string/template to send with request.
-  - **username** (*Optional*): The username for HTTP authentication.
-  - **password** (*Optional*): The password for HTTP authentication.
-  - **timeout** (*Optional*): Timeout for requests. Defaults to 10 seconds.
-  - **content_type** (*Optional*): Content type for the request.
+{% configuration %}
+name:
```
Comment (frenck) [inline source/_components/rest_command.markdown]: The properties below are maps of the device...

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @hmmbob! I've left some comments, could you please take a look? 👍

**Comment by hmmbob** [comment]
Done.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @hmmbob!

Happy Hacktoberfest 🎉

**Pull request merged**